frontend/encore/vuejs.rst

```diff
@@ -40,7 +40,121 @@ updated styles still requires a page refresh.
 
 See :doc:`/frontend/encore/dev-server` for more details.
 
+JSX Support
+-----------
+
+You can enable `JSX with Vue.js`_ by configuring the 2nd parameter of the ``.enableVueLoader`` method:
+
+.. code-block:: diff
+
+    // webpack.config.js
+    // ...
+
+    Encore
+        // ...
+        .addEntry('main', './assets/main.js')
+
+    -     .enableVueLoader()
+    +     .enableVueLoader(() => {}, {
+    +         useJsx: true
+    +     })
+    ;
+
+Afterwards rebuild your frontend assets with Encore. When you do it will show an error message helping you
+install any missing dependencies. After running that command and restarting
+Encore, you're done!
+
+Your ``.jsx`` files will now be transformed through ``@vue/babel-preset-jsx``.
+
+Using styles
+~~~~~~~~~~~~
+
+You can't use ``<style>`` in ``.jsx`` files.
+
+As a workaround, you can import ``.css``, ``.scss``, ``.less`` and ``.styl`` files manually:
+
+.. code-block:: js
```

Comment:
Not sure it is “js” or “JavaScript”

Reply:
Both work

Reply:
Sorry for not being clear, I was on a phone. This was more or less a question to the team.
We prefer „javascript“ here, thank you 😊

Reply:
Oh okay, I will change it ASAP :)

Reply:
Great, thank you very much